src/ui/screens/onboarding/OnboardingDataSlide.qml

```diff
@@ -65,8 +65,9 @@ ColumnLayout {
         leftMargin: 0
         subLabelText: MZI18n.OnboardingDataSlideCheckboxLabel
         showDivider: false
+        isChecked: MZSettings.onboardingDataCollectionEnabled
 
-        onClicked: isChecked = !isChecked
+        onClicked: MZSettings.onboardingDataCollectionEnabled = !MZSettings.onboardingDataCollectionEnabled
```

Comment:
I think we should clear this setting after onboarding is complete, just to be sure. In the unlikely event that the user is put through onboarding again after going through it once. This must always be `false` at the start of onboarding, right?